index.js

```diff
@@ -17,11 +17,52 @@ mongoose
   .then(self => {
     console.log(`Connected to the database: "${self.connection.name}"`);
     // Before adding any documents to the database, let's delete all previous entries
-    return self.connection.dropDatabase();
+    // return self.connection.dropDatabase();
   })
-  .then(() => {
+  .then((result) => {
     // Run your code here, after you have insured that the connection was made
-  })
-  .catch(error => {
-    console.error('Error connecting to the database', error);
-  });
+    const recipe = {
+        "title": "Igor fake recipe",
+        "level": "Amateur Chef",
+        "ingredients": [
+          "1/2 cup rice vinegar",
+          "5 tablespoons honey",
+          "1/3 cup soy sauce (such as Silver Swan®)",
+          "1/4 cup Asian (toasted) sesame oil",
+          "3 tablespoons Asian chili garlic sauce",
+          "3 tablespoons minced garlic",
+          "salt to taste",
+          "8 skinless, boneless chicken thighs"
+        ],
+        "cuisine": "Asian",
+        "dishType": "main_course",
+        "image": "https://images.media-allrecipes.com/userphotos/720x405/815964.jpg",
+        "duration": 40,
+        "creator": "Chef LePapu"
+    }
+
+    Recipe.create(recipe)
+        .then((results) => console.log(`Title of the recipe: ${results}`))
+        .catch((saveErr) => console.error(`Save failed: ${saveErr}`));
+    
+      Recipe.insertMany(data);
+      Recipe.find({});
+
+      Recipe.deleteOne({ title: "Carrot Cake" })
+      .then((del) => {
+        console.log(`Deleted: ${del}`)
+      })
+      .catch((err) => {
+        console.log(`Not deleted: ${err}`);
+      })
+
+      Recipe.findOneAndUpdate({ title:"Rigatoni alla Genovese"}, {duration : 100})
+        .then((update) => {
+        console.log(`Updated: ${update}`);
+      })
+        .catch((error) => {
+      console.error(`Error connecting to the database ${error}`);
+      })
```

Comment:
You can chain the promises to have control over the flow of data

```suggestion
    Recipe.create(recipe)
        .then((results) => {
          console.log(`Title of the recipe: ${results}`
          
          return Recipe.insertMany(data);
         }))
         .then((savedRecipes) => {
           console.log(`All recipes saved in the database: ${savedRecipes}`
           
           return Recipe.find({});
         })
         .then((allRecipesFromDB) => {
           console.log(`All recipes in the database: ${allRecipesFromDB}`
           
           return Recipe.deleteOne({ title: "Carrot Cake" });
         }).then((deletedResults) => {
           console.log(`Recipe deleted from database: ${deletedResults}`;
           
           return Recipe.findOneAndUpdate({ title:"Rigatoni alla Genovese"}, {duration : 100});

         })
         .then((updatedResults) => {
           console.log(`Recipe updated: ${updatedResults}`;
         })
        .catch((saveErr) => console.error(`Save failed: ${saveErr}`));
```